metabase/metabase
# Disable password reset button after click in admin

Related to https://github.com/metabase/metabase/issues/38610

### Description

Disables the password reset button after an admin has clicked it once to avoid multiple API requests to reset from going out. 

### How to verify

1. Go to `/admin/people`
2. Open modal to request a password reset
3. Open network tab in developer tools
4. Spam click the reset button really fast
5. Verify that only request went out to our password endpoint

### Demo

https://www.loom.com/share/bd54e28bf1c54f129790571c135ac3d9?sid=ca7f406d-c8de-4990-a766-2c68a69bf7b4

### Checklist
